src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,53 @@
+.nav {
+  &__list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    padding: 0;
+    margin: 0;
+    list-style-type: none;
+  }
+
+  &__list :nth-child(n+2) {
+    margin-left: 20px;
+  }
+
+  &__item {
+    display: flex;
+    height: 60px;
+  }
+
+  &__link {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #000;
+    position: relative;
+    transition: color 0.4s;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+
+  &__link.is-active {
+    color: #00acdc;
+  }
+
+  &__link.is-active:after {
+    content: "";
+    width: 100%;
+    height: 4px;
+    position: absolute;
+    bottom: 0;
+    left: 0;
```

Comment:
do you need this line if the width is 100%?

Reply:
what line exactly?

Reply:
what Kolya is saying is that you don't need `left: 0` here

Reply:
It was sooo obvious lol!